sbin/patchman

```diff
@@ -520,6 +532,9 @@ def collect_args():
     parser.add_argument(
         '-e', '--errata', action='store_true',
         help='Download CentOS errata from https://cefs.steve-meier.de/')
+    parser.add_argument(
+        '-k', '--killhost',
+        help='Delete host (fqdn)')
```

Comment:
Can the help add some information about how globbing is done and maybe warn that multiple hosts may be deleted?

Reply:
Rewritten per request